instat/dlgView.vb

```diff
@@ -120,7 +118,6 @@ Public Class dlgView
         ucrReceiverView.SetMeAsReceiver()
 
         clsOutputWindowFunction.SetPackageName("utils")
-        clsSeparateWindowFunction.AddParameter("title", Chr(34) & ucrSelectorForView.ucrAvailableDataFrames.cboAvailableDataFrames.SelectedItem & Chr(34), iPosition:=1)

```

Comment:
Yeah it has a developer error that, I need to check with Danny how to handle it, because both radio buttons have the same conditions...with a very slight difference that with the available methods it is not possible to have them set properly